developer/src/kmc-generate/src/template/kmn-keyboard/source/keyboard.keyman-touch-layout

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
Doesn't Keyman Developer offer a choice from a few base layouts when creating a keyman-touch-layout file?
Would that be a future kmc-generate option?

Reply:
Again, a future option, perhaps.